invisible_cities/cities/detsim_simulate_electrons.py

```diff
@@ -0,0 +1,52 @@
+import numpy  as np
+
+from typing    import   List, Tuple, Callable
+
+#######################################
+######### ELECTRON SIMULATION #########
+#######################################
+def generate_electrons(energies    : np.array,
```

Comment:
I'd maybe rename to `generate_ionisation_electrons` (or spelling with z, I think US standard is IC standard normally) just to be clear.